Exiled.API/Features/Player.cs

```diff
@@ -1809,6 +1809,18 @@ public void TrySetCustomRoleFriendlyFire(Dictionary<string, Dictionary<RoleTypeI
         /// <returns> Whether or not the item was able to be added. </returns>
         public bool TryRemoveFriendlyFire(RoleTypeId role) => FriendlyFireMultiplier.Remove(role);
 
+        /// <summary>
+        /// Tries to remove <see cref="RoleTypeId"/> from FriendlyFire rules.
+        /// </summary>
+        /// <param name="roles"> Roles to remove from friendly fire rules. </param>
+        public void TryRemoveFriendlyFire(List<RoleTypeId> roles)
+        {
+            foreach(RoleTypeId roleType in roles)
+            {
+                FriendlyFireMultiplier.Remove(roleType);
+            }
+        }
```

Comment:
```suggestion
        public bool TryRemoveFriendlyFire(IEnumerable<RoleTypeId> roles) => roles.All(TryRemoveFriendlyFire);
```

Reply:
honestly I think using a foreach here just is a bit less confusing and cleaner

Reply:
A method using the prefix "Try" should return a bool value to indicate whether or not the method has been executed successfully. 
`roles.All(TryRemoveFriendlyFire);` will return a bool value to indicate if all roles have been removed using the previous method Player::TryRemoveFriendlyFire(RoleTypeId). 
If you want it to be cleaner, you could also do `roles.All(role => TryRemoveFriendlyFire(role));`